mkdocs.yml

```diff
@@ -29,7 +29,8 @@ nav:
       - Multi-omic Data:
         - IntOmix: Apps/Multi-omic Data/IntOmix.md
       - Data studio:
-        - Data Studio Application: Datastudio.md
+        - Introduction: Datastudio.md
+        - CCLE correlation analysis: ccle.md
```

Comment:
@Shreya-crypto Can you change the file names to "Data Studio.md" and "CCLE Correlation Analysis.md"? And uppercase C and A for the page name for CCLE correlation analysis.

Reply:
And uppercase S for the section name; Data studio